client/lib/analytics/ad-tracking/record-order.ts

```diff
@@ -590,29 +589,6 @@ function recordOrderInJetpackGA(
 			cartToGaPurchase( String( orderId ), cart, wpcomJetpackCartInfo ),
 			Ga4PropertyGtag.JETPACK
 		);
-
-		const jetpackParams = [
-			'event',
-			'purchase',
-			{
-				send_to: TRACKING_IDS.jetpackGoogleAnalyticsGtag,
-				value: wpcomJetpackCartInfo.jetpackCostUSD,
-				currency: 'USD',
-				transaction_id: orderId,
-				coupon: cart.coupon?.toString() ?? '',
-				items: wpcomJetpackCartInfo.jetpackProducts.map(
-					( { product_id, product_name_en, cost, volume } ) => ( {
-						id: product_id.toString(),
-						name: product_name_en.toString(),
-						quantity: parseInt( String( volume ) ),
-						price: ( costToUSD( cost, cart.currency ) ?? '' ).toString(),
-						brand: GA_PRODUCT_BRAND_JETPACK,
-					} )
-				),
-			},
-		];
-		debug( 'recordOrderInJetpackGA: Record Jetpack Purchase', jetpackParams );
```

Comment:
This involves GA4, which we need to keep. We only want to get rid of the calls to GA3/UA.

Reply:
@gmovr jetpackparms is using jetpackGoogleAnalyticsGtag: 'UA-52447-43' as the send to id. Isn't 'UA-52447-43' an old UA ? It is a similar case with akismetParams below.